### PR TITLE
removed the endpoints dictionary

### PR DESCRIPTION
**Pull request by namrata270998**
# Description of change

- removed the endpoints dictionary
- added an Endpoints Class

# Manual QA steps
 - Check if the endpoint API URL is correct for each stream.

 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_pendo/streams.py [hunk 1]
```diff
@@ -21,147 +21,18 @@
...
-        "endpoint": "/api/v1/aggregation",
-    },
-    "guide_events": {
-        "method": "POST",
-        "endpoint": "/api/v1/aggregation",
-    },
-    "feature_events": {
-        "method": "POST",
-        "endpoint": "/api/v1/aggregation",
-        "data": {
```
Comment (dbshah1212) [inline tap_pendo/streams.py:58]: Where did you mention this data? As I can't see the data field in the Endpoints class.

Comment (namrata270998) [reply tap_pendo/streams.py:58]: The 'data' key was not being used anywhere in the code, there was a get_body function that was being used. So haven't included that attribute in the class

File: tap_pendo/streams.py [hunk 2]
```diff
@@ -532,6 +424,7 @@ class Accounts(Stream):
     replication_key = "lastupdated"
     key_properties = ["account_id"]
     method = "POST"
+    # endpoint = Endpoints("/api/v1/account/{accountId}", "GET")
```
Comment (dbshah1212) [inline tap_pendo/streams.py:427]: Do we need this code?

Comment (namrata270998) [reply tap_pendo/streams.py]: No we dont need it, so removed it.

**Review by dbshah1212 — APPROVED** [review]

**Pull request closed without merging**